scheduler/connection_types.py

```diff
@@ -1,14 +1,19 @@
 from typing import Union, Dict, Tuple, Type
 
 import redis
-import valkey
+try:
+    import valkey
+except ImportError:
+    valkey = redis
+    valkey.Valkey = redis.Redis
 
 from scheduler.settings import Broker
 
 ConnectionErrorType = Union[redis.ConnectionError, valkey.ConnectionError]
 ConnectionType = Union[redis.Redis, valkey.Valkey]
 PipelineType = Union[redis.client.Pipeline, valkey.client.Pipeline]
 RedisSentinel = redis.sentinel.Sentinel
+ValkeySentinel = valkey.sentinel.Sentinel
```

Comment:
Maybe have `SentinelType = Union[...]` like `ConnectionType` and `ConnectionErrorType`

Reply:
@cunla now i'm thinking this could also work

```
broker_client = {
    "valkey": "valkey",
    "redis": "redis",
    ...
}

Sentinel = import_string(f"{broker_client[SCHEDULER_CONFIG.BROKER.value]}.sentinel.Sentinel"}
```
same can be done for errors and...

tho it might need modification if a new broker is added that has different structure

let me know what you think